plugins/Denizen/scripts/movement_skillz/double_jump.dsc

```diff
@@ -0,0 +1,68 @@
+# Double jump!!!!!!!!!!!!! But you actually have to jump three times because fuck you
+# You can only jump in water if you touch the ground because it doesn't trigger the player jumps event otherwise :P
+
+
+
+# Flags:
+# capable_to_double_jump:
+#   Without this flag, you cannot double jump. It's pretty much here just to prevent everyone from doing it lel
+#   Can be added in specific scenarios or toggled with the /toggledoublejump command.
+#
+# can_double_jump:
+#   This flag is for when the player just jumped and is eligible to perform a double jump.
+
+
+# Permissions:
+# chaozen.toggledoublejump:
+#   This is the only permission in this script, it's for the command to toggle the 'capable_to_double_jump' flag at the user's will.
```

Comment:
this server literally doesn't have a permission plugin, and all players would have or not have the permission in all cases. use flags instead if you feel like this is needed